### ISSUE TITLE
[Accessibility], WCAG 1.4.3. The combination of white and gray-blue is too low in contrast for normal-sized text

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1961]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/fi/ 

## Description of the problem
Issue: The contrast of white texts against a blue-gray background is a little too low; 4,27:1. Element has insufficient color contrast of 4.27 (foreground color: #f8f9fa, background color: #5d7999, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio is at least 4.5:1
Severity: Minor
Priority: Medium

Fix: Use foreground color: #ffffff and the original background color: #5d7999 to meet a contrast ratio of 4.5:1, and for the implementation to be in accordance with the Style Guide

## Additional information (e.g. screenshots) about the problem

<img width="1004" height="613" alt="Image" src="https://github.com/user-attachments/assets/73c9a494-fd3a-477a-addb-e2883a5db379" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
The issue is not fixed yet.
